lib/tinybucket/model/repository.rb

```diff
@@ -118,6 +118,39 @@ def commit(revision, options = {})
         commits_resource.find(revision, options)
       end
 
+      # Get the list of hooks on this repository.
+      #
+      # @param options [Hash]
+      # @return [Tinybucket::Resource::Hooks]
+      def hooks(options = {})
+        hooks_resource(options)
+      end
```

Comment:
Ooops this is by mistake. I will revisit my branch and remove the unnecessary code